obsei/postprocessor/interface_aggregator.py

```diff
@@ -0,0 +1,102 @@
+from obsei.postprocessor.base_postprocessor import (
+    BasePostprocessorConfig,
+    BasePostprocessor,
+    TextPayload,
+)
+from typing import Any, List, Optional
+
+
+class InterfaceAggregatorConfig(BasePostprocessorConfig):
+    aggregation_method = "most_common"
+
+    def __init__(self, **data):
+        super().__init__(**data)
+
+
+class InterfaceAggregator(BasePostprocessor):
+    def postprocess_input(
+        self, input_list: List[TextPayload], config: InterfaceAggregatorConfig, **kwargs
+    ) -> List[TextPayload]:
+        prev_document_id: int = -1
+        document = []
+        aggregated_payloads = []
+        for output in input_list:
```

Comment:
Let's move this to separate method.
This method will aggregate List of TextPayload to Dict[str, TextPayload] based on given input key.
For example this signature -
```
def payload_aggregator(input_list: List[TextPayload], key: str = "document_id") -> Dict[str, List[TextPayload]]:
```

Reply:
Also what do you feel TextSplitter adding `chunk_id` in the meta and here we sort resultant List based on chunk_id?
It would we become useful util function in this case

Reply:
> @lalitpagaria , Hi , can you have a look at this merge functions . The typecheck is failing because of TextPayload.meta or BasePayload,meta .
> 
> Kindly suggest changes if any , and some suggestion on how to handle the type check (mypy )

I think soon we need to standarize meta as well. But for time being let's do hackish way (please add `#TODO Properly fix this` comment there)

Before loop you can do this -
```
meta = vars(output.meta)
```